### ISSUE TITLE
Dish not showing 

### ISSUE THREAD

**Issue by sagnik3788** [sagnik3788/FlavorFleet#58]

**Comment by sagnik3788** [comment]
![Screenshot from 2024-01-06 15-15-41](https://github.com/sagnik3788/FlavorFleet/assets/116512372/6076006c-1cf5-4cf9-9139-3a703d719013)

**Comment by Gungun2310** [comment]
Hey @sagnik3788 kindly assign this issue to me.

**Comment by sagnik3788** [comment]
@Gungun2310 you can start work on it

**Comment by Gungun2310** [comment]
hey @sagnik3788 is this what you are looking for?
![dishissue](https://github.com/sagnik3788/FlavorFleet/assets/115333744/7246bbce-9d54-4777-b9e9-48708c9a0d90)

**Comment by sagnik3788** [comment]
@Gungun2310 yes but you need to fix the all error of this dish page so that navbar and other elements look like they did before

**Comment by sagnik3788** [comment]
@Gungun2310 any update

**Comment by Gungun2310** [comment]
![Screenshot (70)](https://github.com/sagnik3788/FlavorFleet/assets/115333744/cc1c3be7-9837-4f52-995e-3344457d2d7d)
This works right?

**Comment by sagnik3788** [comment]
Looks good just fix the search and cart allignment then create a pr

**Comment by Gungun2310** [comment]
yeah sure
